### PR TITLE
chore: cow explorer improvements

### LINKED ISSUE #129: chore: improve CoW Explorer Tutorial
https://github.com/cowprotocol/docs-v2/blob/ae8e1316e4db5ff899871d76dbb3824be82f506b/docs/cow-protocol/tutorials/cow-explorer/README.mdx#L12

Should contain examples on:
- [x] How to see the status of my order (how much surplus I got)
- [x] Retrieve the batch I was executed in (other traders, show the graph)
- [x] See partial executions of my limit orders

### PR DESCRIPTION
**Pull request by mfw78**
# Description

Extends the CoW Explorer section.

# Changes

- [x] View by order id (fill-or-kill and partially fillable)
- [x] View by address (show listing of orders)
- [x] View by batch (ie. transaction hash), showing orders and settlement graph

## Related Issues

Fixes #129

### CODE REVIEW

File: docs/cow-protocol/tutorials/cow-explorer/order.md [hunk 1]
```diff
@@ -40,37 +41,30 @@ Below is a simple example of Order ID [`56ac15a9`](https://explorer.cow.fi/order
...
 </details>
 
-A more complex example may be a user who has placed a _partially fillable_ order. This means that the user has expressed a swap intent for a certain amount of tokens, but the order may be filled in multiple batches. This can only be used with `Limit` orders.
+## Partially fillable
+
+A more complex example may be a user who has placed a _partially fillable_ order. This means that the user has expressed a swap intent for a certain amount of tokens, but the order may be filled in multiple batches. This can only be used with [Limit orders](../cow-swap/limit).
 
 ![Partially fillable order overview](/img/explorer/order_partial_overview.png)
 
+By clicking the 'View fills' button, the user can see the fills that have been executed for this order. Fills for the above example are show below:
```
Comment (alfetopito) [inline docs/cow-protocol/tutorials/cow-explorer/order.md]: Typo?

```suggestion
By clicking the 'View fills' button, the user can see the fills that have been executed for this order. Fills for the above example are shown below:
```

File: docs/cow-protocol/tutorials/cow-explorer/order.md [hunk 2]
```diff
@@ -40,37 +41,30 @@ Below is a simple example of Order ID [`56ac15a9`](https://explorer.cow.fi/order
...
-Through Batch Auctions and CoWs (Coincidence of Wants), the protocol can improve on the minimum price that has been shown to the user. That improvement is called _"Surplus"_, and it measures how much better your actual trade price was as opposed to your original limit order price.
-
-#### Prices
-
-As mentioned before, CoW Protocol has the capability of offering users a better price than the limit price that they signed their orders with. Therefore, CoW Protocol has two different prices to take into account, these are:
-
-- **Limit price:** The price set when the order was placed (minus the slippage) & signed. This is the minimum price that you will receive.
-- **Execution price:** The actual price the order was executed at. This parameter can only be equal to or better than the limit price.
+- **Partially-filled:** State for orders that have been partially executed. Parts of the order have been executed and the corresponding funds transferred to the target account.
+- **Pre-signing:** State for orders that are to be pre-signed. Commonly used for orders submitted through the [CoW Swap](../cow-swap) `Safe` interface.
```
Comment (alfetopito) [inline docs/cow-protocol/tutorials/cow-explorer/order.md]: I would mention it's the case for any SC wallet

```suggestion
- **Pre-signing:** State for orders that are to be pre-signed. Commonly used for orders submitted from a Smart Contract wallet, such as the [Safe](https://safe.global/) wallet.
```

File: docs/cow-protocol/tutorials/cow-explorer/order.md [hunk 3]
```diff
@@ -40,37 +41,30 @@ Below is a simple example of Order ID [`56ac15a9`](https://explorer.cow.fi/order
...
+- Total surplus (in this case 0.39% / 860.894 DAI)
+- The limit price applied across all fills (in this case 0.22222222 ANT for DAI)
 
-As CPE is meant to help users visualize meta transactions (signed orders), a critical parameter to show is the State of the signed order. This parameter has the following status:
+## States
 
-- **Open** (pending)**:** State for a standing order. Orders in this state are being considered by the solvers. It is the entry state for all orders, the default from the moment the order is placed. From here the order can transition to all other states
+As CoW Explorer is meant to help users visualize their signed orders, a critical parameter to show is the _state_ of the signed order. This parameter has the following status:
+
+- **Open** (pending)**:** State for a standing order. Orders in this state are being considered by the solvers. It is the entry state for all orders, the default from the moment the order is placed. From here the order can transition to all other states.
```
Comment (alfetopito) [inline docs/cow-protocol/tutorials/cow-explorer/order.md]: Would be worth to add a note about pre-signing orders?
For those types of orders, `Open` is not the entry state.

(Note, I don't know how to create foot notes, this is just a suggestion on the content)

```suggestion
- **Open** (pending)**:** State for a standing order. Orders in this state are being considered by the solvers. It is the entry state for all orders*, the default from the moment the order is placed. From here the order can transition to all other states.

^* Except for pre-sign orders, where the first state is [Pre-signing](link to pre-sign entry)
```

File: docs/cow-protocol/tutorials/cow-explorer/address.md [hunk 4]
```diff
@@ -0,0 +1,31 @@
...
+# By user address
+
+A user may want to see the details of all orders that they have placed. When searching for their address, they will be presented with an overview of all orders that they have placed.
+
+For example, below the user has searched for their address [`0x2557...3af7`](
+https://explorer.cow.fi/gc/address/0x2557ed03e34f0141722a643589f007836a683af7) that contains multiple orders that have been placed on CoW Protocol on Gnosis Chain:
+
+![User address overview](/img/explorer/address_overview.png)
+
+By default, the explorer will show all orders that have been placed by the user, regardless of their status, in reverse chronological order. The user can filter the results by status, and sort the results by any of the columns.
```
Comment (alfetopito) [inline docs/cow-protocol/tutorials/cow-explorer/address.md]: > The user can filter the results by status, and sort the results by any of the columns.

Not really.
Not possible to filter nor sort in this view.

```suggestion
By default, the explorer will show all orders that have been placed by the user, regardless of their status, in reverse chronological order.
```

File: docs/cow-protocol/tutorials/cow-explorer/batch_auction.md [hunk 5]
```diff
@@ -0,0 +1,30 @@
...
+![Batch Orders](/img/explorer/batch_orders.png)
+
+## Settlement graph
+
+This is perhaps one of the most interesting areas, where one can take what was previously an abstract concept of a batch auction, and visualise it in a way that is easy to understand.
+
+This is a visual representation of the routing that was used to settle the batch auction. The graph shows the orders that were settled in the batch auction, and the routing that was used to settle the batch auction.
+The graph is interactive, and the user can hover over any of the edges to see the amount that was settled between the respective nodes of the graph.
+
+![Batch Graph](/img/explorer/batch_graph.png)
```
Comment (alfetopito) [inline docs/cow-protocol/tutorials/cow-explorer/batch_auction.md]: I suggest mentioning the 2 types of visualization: transfer based and trade based, with a picture for each and a brief explanation of what each shows.

Also, why not using the `<Image>` component here like in other places?

**Review by alfetopito — COMMENTED** [review]
A few suggestions

File: docs/cow-protocol/tutorials/cow-explorer/address.md (hunk 4 above)
Comment (mfw78) [reply docs/cow-protocol/tutorials/cow-explorer/address.md]: Thanks for picking up the deliberate mistake :sweat_smile:

File: docs/cow-protocol/tutorials/cow-explorer/batch_auction.md (hunk 5 above)
Comment (mfw78) [reply docs/cow-protocol/tutorials/cow-explorer/batch_auction.md]: As the expert CoW Explorer aficionado, may I kindly request a PR on top of this one, covering the 2 types of visualizations?

File: docs/cow-protocol/tutorials/cow-explorer/order.md (hunk 3 above)
Comment (mfw78) [reply docs/cow-protocol/tutorials/cow-explorer/order.md]: Footnote added. Unfortunately can't link easily back up, but given that it's so close, should be OK.

File: docs/cow-protocol/tutorials/cow-explorer/batch_auction.md (hunk 5 above)
Comment (alfetopito) [reply docs/cow-protocol/tutorials/cow-explorer/batch_auction.md]: There you go https://github.com/cowprotocol/docs-v2/pull/251

**Review by alfetopito — APPROVED** [review]

**Pull request merged**